db/secondbase/structure.sql

```diff
@@ -3,21 +3,35 @@ SET lock_timeout = 0;
 SET idle_in_transaction_session_timeout = 0;
```

Comment:
Not sure if we should include this migration? @simaob

Reply:
@weberjavi , what version of postgres are you running locally? If you didn't change the migrations, then maybe don't include the changes to this file! =)

Reply:
I'm using version `9.6.0.0`
I did not changed the migrations, so I will revert those changes.